### DISCUSSION TITLE
Error-Loading_Remote_Context_Failed

### DISCUSSION THREAD

**Discussion by tnolden** [eclipse-edc/Samples#65] [Q&A]
I get the following error message when trying to run Sample 6:

![image](https://github.com/eclipse-edc/Samples/assets/119170387/d3744103-0152-42dd-8ac4-49311fb71648)

[...]

![image](https://github.com/eclipse-edc/Samples/assets/119170387/2d9c09a8-dcf5-435c-b9be-5b10568efbbc)

This is the mentioned query of Step 3: 
![image](https://github.com/eclipse-edc/Samples/assets/119170387/1a118c4d-ecca-4e5c-86e8-49e7ebfb1613)

So i guess there's a problem with the context "http://www.w3.org/ns/odrl.jsonld". 

Can anybody help me?

**Comment by tnolden** [comment]
Or maybe with the context "https://w3id.org/edc/v0.0.1/ns/", since that side isn't available, as far as i see.

**Comment by abel-nttdata** [comment]
If you remove the line **"@context":"http://www.w3.org/ns/odrl.jsonld",** it works.

But it is a workaround only. :-(

**Comment by ndr-brt** [comment]
this should have been fixed in `main` branch, please check again

**Reply by abel-nttdata** [reply]
Thanks!!
